level2/variables.tf

```diff
@@ -0,0 +1,5 @@
+
+variable "env_code" {
+  type = string
+}
+
```

Comment:
Please remove all newlines at the beginning and have only one newline at the end